.github/workflows/nessie.yaml

```
@@ -475,7 +475,7 @@ jobs:
         working-directory: test/spark
         run: docker-compose up -d
       - name: Copy repository ref
-        run: aws s3 cp --recursive s3://nessie-system-testing/test-data s3://nessie-system-testing/${{ github.run_number }}-spark3-metaclient
+        run: aws s3 cp --recursive s3://nessie-system-testing-data/gc-test-data s3://nessie-system-testing/${{ github.run_number }}-spark3-metaclient
```

```suggestion
        run: aws s3 cp --recursive s3://nessie-system-testing-data/golden-files/gc-test-data s3://nessie-system-testing/${{ github.run_number }}-spark3-metaclient
```

(A friend of mine once put all his files at the top level of the bucket.  You cannot imagine what a *horrible* time he had trying to tune S3 lifecycle policies.  Now we put all our objects with some prefix in them, maybe it won't happen to us again?)

---

Sounds familiar, thanks for the advice!